### PR TITLE
BATM-1197 SLP token support

### PR DESCRIPTION
**Pull request by pvyhnal-generalbytes**
1df9dad can be used as a template for adding other SLP tokens

### CODE REVIEW

File: currencies/src/main/java/com/generalbytes/batm/common/currencies/CryptoCurrency.java [hunk 1]
```diff
@@ -79,6 +79,7 @@
     POT("PotCoin"),
     REP("Augur"),
     SMART("SmartCash"),
+    SPICE("Spice"),
```
Comment (pjill) [inline currencies/src/main/java/com/generalbytes/batm/common/currencies/CryptoCurrency.java:82]: Where does this function come and would i need to add a similar function for my custom token?

Comment (pvyhnal-generalbytes) [reply currencies/src/main/java/com/generalbytes/batm/common/currencies/CryptoCurrency.java:82]: Please see commit https://github.com/GENERALBYTESCOM/batm_public/pull/336/commits/2bce99cc64cb427dd0c9bc98000686ad779b8d91
You can add new SLP tokens in the same way.

If you want to see how it works see other commits in this pull request: https://github.com/GENERALBYTESCOM/batm_public/pull/336/files
Especially
- ElectronCashSlpWallet.java
- SlpPaymentSupport.java

Comment (duranhumes) [reply currencies/src/main/java/com/generalbytes/batm/common/currencies/CryptoCurrency.java:82]: So a new slp token can be added by doing this `SPICE("new-slp-token")`?

Or does adding the name to one of the xml lists some how creates the available function?

Comment (pvyhnal-generalbytes) [reply currencies/src/main/java/com/generalbytes/batm/common/currencies/CryptoCurrency.java:82]: `CryptoCurrency` and `SlpToken` are java enums, you need to add your token to those enums in the same way the SPICE token was added in this commit.

**Pull request merged**